IdentityCore/src/webview/embeddedWebview/challangeHandlers/MSIDClientTLSHandler.m

```diff
@@ -81,8 +81,9 @@ + (BOOL)handleWPJChallenge:(NSURLAuthenticationChallenge *)challenge
                    context:(id<MSIDRequestContext>)context
          completionHandler:(ChallengeCompletionHandler)completionHandler
 {
-    MSIDRegistrationInformation *info = [MSIDWorkPlaceJoinUtil getRegistrationInformation:context error:nil];
-    if (!info || ![info isWorkPlaceJoined])
+    NSError *error = nil;
+    MSIDRegistrationInformation *info = [MSIDWorkPlaceJoinUtil getRegistrationInformation:context urlChallenge:challenge error:&error];
+    if (!info || error)
```

Comment:
would there ever be a case where info == nil && error != nil?

Reply:
Updated the code to just check for info and only log in case of error as we do not want to the leave the developer in an unrecoverable state.